codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -1687,4 +1687,20 @@ public final String getAsString(CharSequence name) {
     public boolean contains(CharSequence name, CharSequence value, boolean ignoreCase) {
         return contains(name.toString(), value.toString(), ignoreCase);
     }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder("HttpHeaders(");
```

Comment:
I think we should better use some "sane" initialCapacity here and then just call `append`.

Reply:
ok